other-cel/restrict-escalation-verbs-roles/restrict-escalation-verbs-roles.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-escalation-verbs-roles
+  annotations:
+    policies.kyverno.io/title: Restrict Escalation Verbs in Roles in CEL expressions
+    policies.kyverno.io/category: Security in CEL 
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Role, ClusterRole, RBAC
+    kyverno.io/kyverno-version: 1.11.0
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno.io/kubernetes-version: "1.26-1.27"
+    policies.kyverno.io/description: >-
+      The verbs `impersonate`, `bind`, and `escalate` may all potentially lead to
+      privilege escalation and should be tightly controlled. This policy prevents
+      use of these verbs in Role or ClusterRole resources.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+    - name: escalate
+      match:
+        any:
+        - resources:
+            kinds:
+              - Role
+              - ClusterRole
+      validate:
+        cel:
+          variables:
+            - name: apiGroups
+              expression: "['*', 'rbac.authorization.k8s.io']"
```

Comment:
Do we need the `*` in this list? I think we could remove it.
Same question for `resources` and `verbs`.

Reply:
We need `*` in `apiGroups`, `resources` and `verbs` to handle an edge case like this:

```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: badcr04
rules:
- apiGroups: ["*"]
  resources: ["*"]
  verbs: ["*"]
```